lib/modules/manager/mix/extract.ts

```diff
@@ -64,17 +67,39 @@ export async function extractPackageFile(
           }
         }
 
-        deps.push(dep);
+        deps.set(app, dep);
```

Comment:
can there be duplicates? if technically possible then log a warning

Reply:
No, there cannot be multiple locked versions. It's one of the issues with the previous implementation, and why I had to change the `mix.exs` fixture to remove duplicates.